repo/packages/O/omsagent/0/package.json

```diff
@@ -0,0 +1,20 @@
+{
+ "packagingVersion": "2.0",
+ "name": "msoms",
+ "version": "1.0.0",
```

Comment:
We have a special tag for universe that contains mesosphere specific omsagent image.
“Version : universe-1.0.0” doesn’t look good, we will revisit this later and fix this.